libpod/networking.go

```diff
@@ -102,6 +114,19 @@ func (r *Runtime) teardownNetNS(ctr *Container) error {
 		return nil
 	}
 
+	// Because we are using iptables to allow the container to resolve DNS
+	// on per IP address, we also need to try to remove the iptables rule
+	// on cleanup. Remove when https://github.com/containernetworking/plugins/pull/75
+	// is merged.
+	for _, ip := range ctr.state.IPs {
+		logrus.Info("Adding Iptables run for %s", ip.Address.IP.String())
+		iptables := []string{"iptables", "-t", "filter", "-D", "FORWARD", "-s", ip.Address.IP.String(), "!", "-o", ip.Address.IP.String(), "-j", "ACCEPT"}
```

Comment:
Can you logrus.Debugf the arguments we're passing to IPTables?

Reply:
done